### PR TITLE
Add Geist Typeface to Project

### PR DESCRIPTION
**Pull request by SHADOW-LIGHTS**
### Description
Integrated the Geist typeface into the project to enhance readability and design consistency. This addition aims to improve the visual experience for both developers and users.

### Related Issue
#196 

### Changes Made
- Added Geist Sans variant for versatile design use.
- Included Geist monospace variant for improved code readability.
- Updated Tailwind config and CSS files to apply the new typeface across the project.

### Screenshots
**Now -**
image
**Before -**
image

### Checklist

- [x] I have tagged the issue in this PR.
- [x] I have attached necessary screenshots.
- [x] I have provided a short description of the PR.
- [x] I ran `yarn build` and build is successful
- [x] My code follows the style guidelines of this project.
- [x] I have added necessary documentation (if applicable)

### CODE REVIEW

**Comment by SHADOW-LIGHTS** [comment]
This font truly goes with the story of the company. While images might not fully capture the impact of this change, the improved readability and design consistency will surely enhance the user experience. 
It’ll blend seamlessly with all the future changes and fit perfectly with other elements.

File: app/layout.tsx [hunk 1]
```diff
@@ -32,10 +20,10 @@ export default function RootLayout({
   children: React.ReactNode;
 }) {
   return (
-    <html lang="en">
+    <html lang="en" className={`${GeistSans.variable} ${GeistMono.variable}`}>
       <PHProvider>
         <body
-          className={`${inter.variable} ${architects_daughter.variable} font-inter bg-background tracking-tight text-gray-900 antialiased`}
+          className={`--font-geist-sans bg-background tracking-tight text-gray-900 antialiased`}
```
Comment (ICEPrey) [inline app/layout.tsx:26]: This is wrong we should just use the tailwind class to apply the font
```suggestion
          className={`font-sans bg-background tracking-tight text-gray-900 antialiased`}
```

**Review by ICEPrey — CHANGES_REQUESTED** [review]
minor change

**Comment by SHADOW-LIGHTS** [comment]
> minor change

bad mistake. Corrected it

**Review by ICEPrey — APPROVED** [review]
looks good to me

**Review by Fryingpannn — APPROVED** [review]

**Pull request merged**